Package.resolved

```diff
@@ -50,9 +50,9 @@
         "package": "SwiftGraphQLParser",
         "repositoryURL": "https://github.com/Shopify/SwiftGraphQLParser",
         "state": {
-          "branch": null,
-          "revision": "0fcadd4bf9a348c6af97fed05d1c25302e86e1ec",
-          "version": "0.1.7"
+          "branch": "field-init-public",
+          "revision": "f1280fbebf6e8ff221b24dc7df72d700520acac2",
+          "version": null
```

Comment:
need to update this again once https://github.com/Shopify/SwiftGraphQLParser/pull/2 is merged